CMake/External_OpenCV.cmake

```diff
@@ -41,6 +61,20 @@ else()
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DWITH_FFMPEG=OFF)
 endif()
 
+# Set VTK dependency if we're locally building it
+if(fletch_ENABLE_OpenCV_VTK)
+  message(STATUS "OpenCV depending on fletch VTK")
+  if (OpenCV_version VERSION_LESS 3.2.0 AND VTK_version VERSION_GREATER_EQUAL 7.0)
+    message(FATAL_ERRROR "OpenCV versions before 3.2 can only handle pre 7.0 VTK versions")
+  endif()
+  list(APPEND OpenCV_DEPENDS VTK)
+  list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DWITH_VTK:BOOL=TRUE
+    -DVTK_DIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib/cmake/vtk-${VTK_version}
```

Comment:
At this point, either way is ok. It will be good to test whether we can just enable it when VTK is enabled and be done with it. The catch is, will something in  OpenCV break as a result. The only other downside would be a probable increase in build time.